docs/sim/observations.rst

```diff
@@ -21,20 +21,22 @@ is a Python `NamedTuple` with the following fields:
     * `reached_max_episode_steps` - `True` if the vehicle has reached its max episode steps
 * `ego_vehicle_state` - a `VehicleObservation` `NamedTuple` for the ego vehicle with the following fields:
     * `id` - a string identifier for this vehicle
-    * `position` - A 3D numpy array (x, y, z) of the center of the vehicle bounding box's bottom plane
+    * `position` - A 3D array (x, y, z) of the center of the vehicle bounding box's bottom plane
```

Comment:
`type(position) = numpy.ndarray`
`position.dtype = float64`
`position.shape = (3,)`

```suggestion
    * `position` - Coordinate of the center of the vehicle bounding box's bottom plane. shape=(3,). dtype=np.float64.
```

I think it is not a 3D array.